### DISCUSSION TITLE
Printer config

### DISCUSSION THREAD

**Discussion by laguinch** [photobooth-app/photobooth-app#113] [Q&A]
I have a canon CP1300 printer (same as yours I mean) what is the config to insert and where ?
the command lp -d Canon_SELPHY_CP1300_USB_ -o landscape -o fit-to-page photobooth-data/media/proceed/full/image_True_XXXXXXX.jpg is running -> request id is Canon_SELPHY_CP1300_USB_ -3 (1 file(s))
but if I trie to print I have this error :
`Something went wrong, Exception: print failed, error Command mspaint /p /home/pi/photobooth-data/media/processed/full/image_True_20231028-094941-
087598.jpg returned non-zero exit statut 127` 
Print Service

**Comment by mgineer85** [comment]
So you need to set the command in the config tab -> hardwareinputoutput:
`lp -d Canon_SELPHY_CP1300_USB_ -o landscape -o fit-to-page {filename}`

Currently it seems the default is still set (which is mspaint).
